ludwig/automl/defaults/text/bert_config.yaml

```diff
@@ -1,9 +1,17 @@
 trainer:
-  epochs: 5
+  epochs: 10
   learning_rate_scheduler:
-    warmup_evaluations: 0
+    warmup_fraction: 0.1
+    decay: linear
   optimizer:
     type: adamw
+  use_mixed_precision: true
+
+defaults:
+  text:
+    encoder:
+      type: bert
+      trainable: true
```

Comment:
Why do we default to `trainable: true`? Thinking about an iterative approach where you would:

1. Train default model (parallel CNN)
2. Train a model with pretrained BERT
3. Train a model to fine-tune BERT

Reply:
Agreed in the iterative case. In the "best single shot model" case, I would say that setting `trainable: true` right away makes sense.

Reply:
Agreed in the iterative case. In the "best single shot model" case, I would say that setting `trainable: true` right away makes sense.